common/lib/common-definitions/package.json

```diff
@@ -11,24 +11,71 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
 	"exports": {
 		".": {
-			"types": "./dist/index.d.ts",
-			"default": "./dist/index.js"
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
```

Comment:
common-definitions is all alpha/internal and is completely deprecated. Shouldn't bother with exports update.

Reply:
Changed to type-only exports and made the `main` field `""`